src/app/components/layout/inputs/LimitedTextArea.js

```diff
@@ -75,7 +75,6 @@ LimitedTextArea.defaultProps = {
   value: '',
   setValue: null,
   helpContent: null,
-  textFieldProps: {},
```

Comment:
@dariusk I *think this is correct to remove these things?
The reason is, when they are added, the dom outputs this:

<img width="221" alt="Screenshot 2023-10-13 at 12 16 12 PM" src="https://github.com/meedan/check-web/assets/418001/bacb55f1-2fb9-429e-98c9-44b91ab8ec5a">

Obviously let me know if something different should be done